### PR TITLE
initial

### PR DESCRIPTION
**Pull request by stutisk**
Animation

### CODE REVIEW

File: components/buttons/button.css [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+    font-size: 1.5rem;
+    background: var(--dark-blue-color);
+    text-align: center;
+    text-decoration: none;
+    margin: 5px 2px;
+    color: white;
+}
+.icon-btn:hover{
+    opacity:0.8;
+
```
Comment (FarhanMobashir) [inline components/buttons/button.css:80]: use prettier for code formatting https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode

File: components/buttons/button.css [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+    background: var(--red-color);
+}
+.float-btn{
+    background-color: var(--primary-color);
+    color:white;
+   border-radius:50%;
+   padding: 20px;
+   text-align: center;
+   margin: 4px ;
+   font-size: 1.5rem;
```
Comment (FarhanMobashir) [inline components/buttons/button.css:99]: make some css variable for the sizes of the font thatt you keep reusing

**Review by FarhanMobashir — COMMENTED** [review]
go through the comments and make the change if it makes sense for you. Thank you

**Review by yogesh- — CHANGES_REQUESTED** [review]
Look at the changes suggested in the first review, rest all looks good.

**Comment by stutisk** [comment]
hey ! i have made the changes , do check it once , thanku

File: components/buttons/button.css [hunk 3]
```diff
@@ -0,0 +1,101 @@
+:root {
+    --primary-color:#4A777A;
+    --text-color:white;
+    --text-color-secondry:black;
```
[the remark this answers is not in the document]
Comment (stutisk) [reply components/buttons/button.css]: hey ,i have added the changes , thanku

**Pull request merged**